File: AppHome/app/src/main/java/com/example/goodstart/models/Section.java
package com.example.goodstart.models;

import java.util.List;

public class Section {
    public String id;
    public boolean isHeader;
    public boolean isAliyahHeader;
    public boolean isChapterHeader;
    public String he;
    public String en;
    public String ordinal;
    public List<RashiItem> rashi;
    public Integer verseNum;
    public Integer chapterNum;

    public static class RashiItem {
        public String he;
    }
}

File: AppHome/app/src/main/java/com/example/goodstart/model/Section.kt
package com.example.goodstart.model

data class Section(
    val id: String? = null,
    val isHeader: Boolean = false,
    val isAliyahHeader: Boolean = false,
    val isChapterHeader: Boolean = false,
    val he: String? = null,
    val en: String? = null,
    val ordinal: String? = null,
    val rashi: List<RashiItem>? = null,
    val verseNum: Int? = null,
    val chapterNum: Int? = null
) {
    data class RashiItem(val he: String? = null)
}
